cartography/data/jobs/analysis/gsuite_human_link.json

```diff
@@ -0,0 +1,9 @@
+{
+  "statements": [
+  {
+    "query": "MATCH (human:Human), (guser:GSuiteUser) WHERE human.email = guser.email WITH human, guser MERGE (human)-[r:IDENTITY_GSUITE]->(guser) ON CREATE SET r.firstseen = {UPDATE_TAG} SET r.lastupdated = {UPDATE_TAG}",
```

Comment:
You don't need the WITH clause in relationship statements. It doesn't hurt but it also doesn't help perf. 
![image](https://user-images.githubusercontent.com/14211633/68397153-9302ad80-0127-11ea-8a2f-a032bbf38367.png)